moped-editor/src/views/projects/projectView/ProjectSummary.js

```diff
@@ -110,9 +112,22 @@ const ProjectSummary = () => {
         </Grid>
         <Grid item xs={12} md={6}>
           {project_extent_geojson && project_extent_ids && (
-            <ProjectSummaryMap
+            <>
+              <ProjectSummaryMap
+                selectedLayerIds={project_extent_ids}
+                projectExtentGeoJSON={project_extent_geojson}
+                setIsEditing={setIsEditing}
+              />
+            </>
+          )}
+          {isEditing && (
+            <ProjectSummaryEditMap
+              projectId={projectId}
               selectedLayerIds={project_extent_ids}
               projectExtentGeoJSON={project_extent_geojson}
+              isEditing={isEditing}
+              setIsEditing={setIsEditing}
+              refetchProjectDetails={refetch}
```

Comment:
Just impressive how easy you make it look 👍

Reply:
➕